### ISSUE TITLE
f11 should not be passed through when no remote access connection

### ISSUE THREAD

**Issue by CyrilleB79** [nvaccess/nvda#17874]
### Steps to reproduce:
1. start NVDA
2. press NVDA+F11
3. Go to remote settings, check "Enable Remote Access" and validate with OK
4. Press `NVDA+f11`
5. Start a new connection, as client, and controlling another machine (but do not connect another machine at the other end)
6. Press `NVDA+f11` various times
7. Press NVDA+r to disconnect
8. Start a new connection, as client, and being controlled by another machine (but do not connect the controller yet)
9. Press `NVDA+f11`
### Actual behavior:
2. NVDA says "Action unavailable when Remote Access is disabled" (OK)
4. NVDA says nothing and passes `f11` to the focused application, e.g. full screen command in a browser.
6. NVDA toggles between controlled and local machine, but there is no controlled machine connected
9. NVDA says nothing and passes `f11` to the focused application, e.g. full screen command in a browser.

In all these cases, no action is performed but the result is not consistant: sometimes `f11` is passed through (without NVDA modifier), sometimes a message indicates that the action cannot be performed and sometimes the toggle action is performed even if there is no remote machine connected yet.

Minor other defect:
7. NVDA says "Disconnected" twice.
### Expected behavior:
A clear message should indicate the situation to the user:

2. Already OK
4. When the user press `NVDA+f11`, it's already clear that they do not want to pass `f11` to the app. Instead an error message indicating "No connection currently established" should be issued
6. Until the remote machine is connected, there is no point with toggling the control back and forth between controlled and controller. Thus, I'd better expect a message indicated "Remote machine not connected" to indicate that I cannot toggle key commands to remote machine.
9. As for 4.: a message indicating that this command is unavailable when you are the controlled mahcine should be reported.

Minor other issue:
7. NVDA should say "Disconnected" only once.
#### NVDA installed/portable/running from source:
source
#### NVDA version:
commit e1c942f96a93ac24b14d696e5fc233592684a107
#### Windows version:
10 22H2
#### Does the issue still occur after restarting your computer?
yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
no
#### If NVDA add-ons are disabled, is your problem still occurring?
not tested
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
n/a

**Comment by CyrilleB79** [comment]
@SaschaCowley in some cases, the user can type `NVDA+f11` and gets `f11` passed to the application. It is an undocumented and overall unexpected result.
Wouldn't this issue deserve a p3 for this instead?
